### PR TITLE
chore: update ACM docs

### LINKED ISSUE #1260: "cluster_endpoint is not expected here" message - in ACM v21.0.0
### TL;DR

When deploying ACM using version 21.0.0 it fails as it does not expect cluster_endpoint and operator_path. 
When I comment them out it is not finishing deployment correctly. 
Once I go back to v20.0.0 it complains that the vars are missing. 
Documentation still mentions cluster_endpoint as variable, so I assume this is a bug?

### Expected behavior

installing ACM 

### Observed behavior

Says cluster_endpoint is not expected. 

```
Error: Unsupported argument

  on acm.tf line 11, in module "acm":
  11:   cluster_endpoint = module.gke-west.endpoint

An argument named "cluster_endpoint" is not expected here.


Error: Unsupported argument

  on acm.tf line 12, in module "acm":
  12:   operator_path    = var.operator_path

An argument named "operator_path" is not expected here.
```

### Terraform Configuration

```hcl
module "acm" {
  source           = "terraform-google-modules/kubernetes-engine/google//modules/acm"
  version          = "v21.0.0"
  project_id       = var.gcp_project_id
  location         = module.gke-west.location
  cluster_name     = module.gke-west.name
  sync_repo        = var.acm_sync_repo
  sync_branch      = var.acm_sync_branch
  policy_dir       = var.acm_policy_dir
  create_ssh_key   = var.create_ssh_key
  cluster_endpoint = module.gke-west.endpoint
  operator_path    = var.operator_path
  secret_type      = var.secret_type
  source_format    = var.source_format
  ssh_auth_key     = var.ssh_auth_key
  install_template_library = var.install_template_library
  enable_policy_controller = var.enable_policy_controller
}
```

### Terraform Version

```sh
v0.14.10
```

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by bharathkkb**
fixes #1260

### CODE REVIEW

File: modules/acm/README.md [hunk 1]
```diff
@@ -18,23 +18,39 @@ Other clusters should disable feature activation by setting `enable_fleet_featur
 There is a [full example](../../examples/simple_zonal_with_acm) provided. Simple usage is as follows:
 
 ```tf
+provider "kubernetes" {
+  host                   = "https://${module.gke.endpoint}"
+  token                  = data.google_client_config.default.access_token
+  cluster_ca_certificate = base64decode(module.gke.ca_certificate)
+}
+
```
Comment (apeabody) [inline modules/acm/README.md:26]: Would it be preferable to also have ```data "google_client_config" "default" {}``` here?

**Review by apeabody — APPROVED** [review]
LGTM!

File: modules/acm/README.md (hunk 1 above)
Comment (bharathkkb) [reply modules/acm/README.md:28]: Missed that, thanks!

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Pull request merged**